packages/augur-ui/src/config/network.json

```diff
@@ -27,7 +28,7 @@
   "kovan": {
     "networkId": "42",
     "ethereum": {
-      "http": "https://eth-kovan.alchemyapi.io/jsonrpc/1FomA6seLdWDvpIRvL9J5NhwPHLIGbWA",
+      "http": "",
```

Comment:
We should make this key optional on SDKConfiguration and then remove it from here.